packages/ember-metal/lib/chains.ts

```diff
@@ -191,21 +190,13 @@ class ChainNode {
     this._paths = undefined;
 
     // It is false for the root of a chain (because we have no parent)
-    let isWatching = parent !== null;
-    let object;
-    if (isWatching) {
+    if ((this._isWatching = parent !== null)) {
```

Comment:
I'm not a fan of this pattern, it kinda seems like a mistake if you're reading with full attention.